src/lib.rs

```diff
@@ -2399,6 +2399,44 @@ where
     }
 }
 
+/// Conversion from [IntoIterator] to [StreamingIterator].
+pub trait IntoStreamingIterator: IntoIterator
+where
+    Self: Sized,
+{
+    /// Turns an [IntoIterator] into a [StreamingIterator].
+    ///
+    /// Calling [into_streaming_iter](IntoStreamingIterator::into_streaming_iter) on an [IntoIterator] is equivalent to using [convert].
```

Comment:
Why bother with a self-referencing link on each of these methods? Perhaps just "Calling this..."?

Reply:
Ah, your new commit was not the change I intended. Referencing `IntoIterator` in the first line is good, especially "of references" and "of mutable references" on the other two, but I meant more like this:
```suggestion
    /// Calling this method on an [IntoIterator] is equivalent to using [convert].
```

Reply:
got it